static/styles/popovers.scss

```diff
@@ -1,4 +1,198 @@
+// Move Tooltip + popover css from bootstrap v2.1.1
+// Override css in bootstrap to support
+// our custom changes in tooltip.js.
+.tooltip {
+    position: absolute;
+    z-index: 1030;
+    display: block;
+    padding: 5px;
+    font-size: 11px;
+    opacity: 0;
+    filter: alpha(opacity=0);
+    visibility: visible;
+}
```

Comment:
I think this might end up being more readable/maintainable if we had a `static/third/` file with the stock bootstrap-tooltip CSS, and then this file remained just the custom Zulip CSS.